exchange.c

```diff
@@ -269,7 +269,7 @@ static int domain_exchange_once(int (*layoutfunc)(int p), ExchangePlan * plan)
             if(!SlotsManager->info[ptype].enabled) continue;
 
             if(BASESLOT(i)->ID != P[i].ID) {
```

Comment:
the direct access of `.enabled` looked suspicious is it the same as the macro?

Reply:
It is the same.

Reply:
Yes, but the macro is only in slotsmanager.c